bindings/python/src/tokenizer.rs

```diff
@@ -38,6 +38,13 @@ impl Tokenizer {
         }
     }
 
+    fn num_added_tokens(&self, is_pair: bool) -> PyResult<usize> {
```

Comment:
I'm not sure it really makes sense directly exposed on the `Tokenizer`. I'd expect this to be the number of added tokens that gets returned by `get_vocab_size(with_added_tokens=True)`.

This can simply be retrieved by doing `tokenizer.post_processor.num_added_tokens(is_pair)`

Reply:
I exposed this method on Tokenizer for the case where there is no post_processor. 
In this case, it requires to check if post_processor is not null before accessing the property and returning 0 if null.

Having this wrapped on the Rust side doesn't require such checking to be done in every bindings we would have.

Reply:
I understand. My main concern is just about having `add_tokens` that does something, and `num_added_tokens` just right beside it, doing something completely different. If we find the right name, we can add it here!

Reply:
What about just keeping added_tokens everywhere ?

Reply:
As discussed, let's use `num_special_tokens_to_add`